.github/workflows/copilot-setup-steps.yml

```diff
@@ -17,11 +17,39 @@ jobs:
     steps:
       - uses: actions/checkout@v4.2.2
 
-      - name: Install dependencies
-        run: sudo ./eng/common/native/install-dependencies.sh
-      - name: Restore solution
-        run: ./build.sh --restore --excludecibinarylog --warnaserror false /p:BuildAllConfigurations=true /p:DotNetBuildAllRuntimePacks=true
+      - name: Install Dependencies
+        env:
+          LIBMSQUIC_VERSION: '2.4.8'
+        run: |
+          sudo ./eng/common/native/install-dependencies.sh && \
+          sudo apt-get install -qq -y \
+            curl \
+            libbpf1 \
+            libelf1t64 \
+            libnl-3-200 \
+            libnl-route-3-200 \
+            libnuma1 \
+            libxdp1 && \
+          curl -LO "https://packages.microsoft.com/ubuntu/24.04/prod/pool/main/libm/libmsquic/libmsquic_${LIBMSQUIC_VERSION}_amd64.deb" && \
+          sudo dpkg -i libmsquic* && \
+          rm libmsquic*
```

Comment:
I'd really prefer if we do what was mentioned in https://github.com/dotnet/runtime/issues/115946 to centralize common dependencies and improve the contributor experience too.  Can we create a script `./eng/install-dependencies.sh` that does everything here?

Reply:
I totally agree, but the msquic part is tricky. Let's tackle that in a follow-up